.github/ISSUE_TEMPLATE/midpoint-review.md

```diff
@@ -84,40 +85,22 @@ This may include mockups, prototypes, Github files, Word document or outline, st
 - OCTO-DE Product lead: [Insert name here]
 - OCTO-DE Design lead: Ryan Thurlwell
 
-### Team Members
-<!-- Add anyone from your team who significantly contributed to the artifacts provided, otherwise, delete list -->
-- [Insert name here]
-- [Insert name here]
 
-### VFS Attendees
-<!-- Delete any of the below if they are not applicable for your meeting -->
-- Content and IA: 
-- Accessibility: 
-- QA specialists: 
-
-
-## Scheduling
+## Asynchronous review option
 Would you like this review to be asynchronous? 
 
 Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
 - [ ] Yes
 - [ ] No
 
+## Scheduling
+- Open the [Calendly design intent calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
```

Comment:
Can we add a note here (or above) that teams still need to book via Calendly for an asynchronous Midpoint Review? We can specify that no zoom meeting will be added however the timeslot reservation is necessary in order to manage workload and intake. What do you think?

Reply:
Yes, take a look now